roles/github/templates/generic.yml.j2

```diff
@@ -1,3 +1,12 @@
+<%- if github_environment_type == 'kayobe' -%>
+<%- set github_runs_on = github_runs_on + ['${{ inputs.kayobe_environment }}'] -%>
+<%- set github_registry_url = "${{ vars[format('{0}_REGISTRY_URL', inputs.kayobe_environment)] }}" -%>
+<%- set github_registry_password = "${{ secrets[format('{0}_REGISTRY_PASSWORD', inputs.kayobe_environment)] }}" -%>
+<%- set _ = workflow.update({"concurrency_group": "format('{0}-{1}', " + workflow.concurrency_group + ", '${{ inputs.kayobe_environment }}')" })  -%>
+<%- set _ = github_kayobe_arguments.update({"KAYOBE_ENVIRONMENT": '${{ inputs.kayobe_environment }}'}) -%>
+<%- set _ = github_kayobe_arguments.update({"KAYOBE_AUTOMATION_SSH_PRIVATE_KEY": "${{ secrets[format('{0}_KAYOBE_AUTOMATION_SSH_PRIVATE_KEY', inputs.kayobe_environment)] }}" }) -%>
+<%- set _ = github_kayobe_arguments.update({"KAYOBE_VAULT_PASSWORD": "${{ secrets[format('{0}_KAYOBE_VAULT_PASSWORD', inputs.kayobe_environment)] }}" }) -%>
```

Comment:
Should these go into `github_default_kayobe_arguments` to allow them to be overridden?

Reply:
Not sure what would be overridden maybe the key/name of secret?

I have update `github_kayobe_arguments` rather than `github_default_kayobe_arguments` to enforce priority.

Reply:
Change now allows a user to enforce shared secrets/variables between environments.